### ISSUE TITLE
Add an example map

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shiny-template#82]
**Is your feature request related to a problem? Please describe.**
People like maps. We can do maps. DAC commented on maps. We should show how to do maps.

Notes:
- Ensure the colours in the legend have a black outline to give a suitable contrast.
- Check keyboard navigability and focus indicator.
- Ensure it's hidden from screen reader and they can use the table behind it instead
- Think about whether there is a general function we could put into dfeshiny that we could use for the core styling we'd want all maps to have.

**Describe the solution you'd like**
An example map in the tool, showing some data at local authority level, with a dropdown to change the data. Potentially a slide for year if we have multiple too. 

Ideally as a bslib card with a table and download options behind it.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by cjrace** [comment]
Should copy the EES approach to adding a border around the legend colours from here - https://github.com/dfe-analytical-services/explore-education-statistics/pull/5328
